src/streamlit_extras/mention/__init__.py

```diff
@@ -65,10 +62,11 @@ def mention(label: str, url: str, icon: str = "🔗", write: bool = True):
         span(),
     )
 
+    html = STYLE_HTML + str(mention_html)
```

Comment:
Adding style inline with content, so that style doesn't take vertical space anymore!